### PR TITLE
Allow CSS class to be set for table header.

### PR DESCRIPTION
**Pull request by kjarri**
I wasn't able to set the class for thead, only for the cells via the theadCell property.

### CODE REVIEW

File: tests/dummy/app/routes/examples/custom-messages.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export default ExampleRoute.extend({
   setupController(controller) {
     this._super(...arguments);
     set(controller, 'themeInstance', BootstrapTheme.create({
+      thead: '',
```
Comment (onechiporenko) [inline tests/dummy/app/routes/examples/custom-messages.js]: Is it needed here?

**Comment by onechiporenko** [comment]
@kjarri, thanks for PR! I have only one question. Other LGTM.

File: tests/dummy/app/routes/examples/custom-messages.js (hunk 1 above)
Comment (kjarri) [reply tests/dummy/app/routes/examples/custom-messages.js]: Nope, it is not. I just put it there for testing purposes and accidentally included the file in the commit.

**Comment by onechiporenko** [comment]
Thanks!

**Pull request merged**